spaces/S000166/README.md

```diff
@@ -0,0 +1,6 @@
+---
+uid: S000166
+name: Left ray topology on $\omega+1$
+---
+
+$X$ has underlying set $\omega+1$. The open sets are exactly $X$ and $[0,a)$ for $a \in X$. Equivalently, this is the Alexandrov topology for the reverse ordering on $\omega+1$.
```

Comment:
```suggestion
$X$ has underlying set $\omega+1$. The open sets are exactly $X$ and $[0,a)$ for $a \in X$. This is the Alexandrov topology for the reverse ordering on $\omega+1$.
```

Reply:
I wrote "equivalently" here mostly because it is not immediately obvious, since Alexandrov topology, by definition, would have $[0,a]$ as open basis instead of $[0, a)$. Perhaps a better way is say "Equivalently, the topology is generated by the open basis $[0,a]$ for $a \in X$. This is the Alexandrov topology..."?

Reply:
When I made the suggestion I mistakenly thought it was immediately obvious. I do like the following though:

>  "Equivalently, the open sets are exactly downwards closed subsets of 
X. This is the Alexandrov topology..."?

And now I agree the Alexandrov trait file should state 'By definition.', since it's an equivalent form in pi-base.